app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cmd = command.split(" ")
+    if cmd[1] == cmd[2]:
+        pass
+
+    f1 = open(cmd[1], "r")
+    f2 = open(cmd[2], "w")
```

Comment:
what is cmd[1], cmd[2]?
create a separated variables for them and call them on demand